src/doomtype.h

```diff
@@ -138,5 +138,12 @@ typedef int16_t dpixel_t;
 
 #define arrlen(array) (sizeof(array) / sizeof(*array))
 
+#ifndef MIN
```

Comment:
Won't these clash on WIN32? We generally should prefer macros "namespaced" (as fas as C will allow) to our own namespace.

Reply:
I don't think that Windows has all-upper-case MIN/MAX, at least I don't see any clash in other projects which introduce them. Anyway, they are only used in two cases here, so I'd rather spell them out.